demo/custom.html

```diff
@@ -0,0 +1,339 @@
+
+<h1>Standard Table</h1>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>Column 1</th>
+      <th>Column 2</th>
+      <th>Column 3</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td data-label="Column 1">Row A</td>
+      <td data-label="Column 2">Cell A2</td>
+      <td data-label="Column 3">Cell A3</td>
+    </tr>
+    <tr>
+      <td data-label="Column 1">Row B</td>
+      <td data-label="Column 2">Cell B2</td>
+      <td data-label="Column 3">Cell B3</td>
+    </tr>
+    <tr>
+      <td data-label="Column 1">Row C</td>
+      <td data-label="Column 2">Cell C2</td>
+      <td data-label="Column 3">Cell C3</td>
+    </tr>
+    <tr>
+      <td data-label="Column 1">Row D</td>
+      <td data-label="Column 2">Cell D2</td>
+      <td data-label="Column 3">Cell D3</td>
+    </tr>
+  </tbody>
+</table>
+
+
+<h1>Standard Table with striped rows</h1>
```

Comment:
Should this lose it's stripes on small screens? I'm assuming they're gone because there's no small variation here, but just wanted to be sure. If this is correct, can you call out that stripes only exist` > 600px`

<img width="341" alt="screen shot 2015-10-08 at 10 25 29 am" src="https://cloud.githubusercontent.com/assets/1280430/10369293/761fbd06-6da7-11e5-9d96-476be9b583d7.png">